okhttp-clients/src/main/java/com/palantir/conjure/java/okhttp/InstrumentedInterceptor.java

```diff
@@ -37,11 +38,17 @@
     private final HostEventsSink hostEventsSink;
     private final String serviceName;
     private final Timer responseTimer;
+    private final Meter ioExceptionMeter;
 
     InstrumentedInterceptor(TaggedMetricRegistry registry, HostEventsSink hostEventsSink, String serviceName) {
         this.hostEventsSink = hostEventsSink;
         this.serviceName = serviceName;
         this.responseTimer = registry.timer(name());
+        this.ioExceptionMeter = registry.meter(MetricName.builder()
+                .safeName("client.response.failure")
```

Comment:
We're estimating this will cost $5875/month right?

Reply:
Given that we already have `server.response.error` would we do a little `s/failure/error/` to be consistent

Reply:
Discussed with @dansanduleac the other day, but we may want to consider coupling this with a conjure-java-runtime-api change for client exceptions which didn't produce a response status. This way we can differentiate socket timeout, connection timeout, no route to host, etc. Ideally we would spec out the client runtime exceptions we throw from clients so that retrofit, dialogue, and feign produce the same failures, and migration between them has fewer subtle edge cases.

Reply:
My estimate is $8k/mo

Reply:
Agree it would be good to differentiate between types of IOExceptions down the line. I don't see that as a blocker for merging this PR though.

Reply:
I don’t think this pr makes it more difficult to implement that in the future :-)